src/psycopmlutils/loaders/raw/load_diagnoses.py

```diff
@@ -298,39 +298,39 @@ def f1_disorders(n: Optional[int] = None) -> pd.DataFrame:
             n=n,
         )
 
-    @data_loaders.register("alcohol and tobacco dependencies")
+    @data_loaders.register("alcohol_and_tobacco_dependencies")
     def alcohol_and_tobacco(n: Optional[int] = None) -> pd.DataFrame:
         return LoadDiagnoses.from_physical_visits(
             icd_code=["f10", "f17"],
             wildcard_icd_code=True,
             n=n,
         )
 
-    @data_loaders.register("opioid and sedative dependencies")
+    @data_loaders.register("opioid_and_sedative_dependencies")
     def opioids_and_sedatives(n: Optional[int] = None) -> pd.DataFrame:
         return LoadDiagnoses.from_physical_visits(
             icd_code=["f11", "f13"],
             wildcard_icd_code=True,
             n=n,
         )
 
-    @data_loaders.register("cannabinoid and hallucinogen dependencies")
+    @data_loaders.register("cannabinoid_and_hallucinogen_dependencies")
     def cannabinoids_and_hallucinogens(n: Optional[int] = None) -> pd.DataFrame:
         return LoadDiagnoses.from_physical_visits(
             icd_code=["f12", "f16"],
             wildcard_icd_code=True,
             n=n,
         )
```

Comment:
Are there good arguments for having these collected in the same columns? I'd recommend splitting them up in the loader functions (e.g. `cannbinoid_dependency` and `hallucinogen_dependency` separately), and then we can choose to join them during pre-processing :-)

Reply:
Not really :) So good point! I'll split them up, commit and then merge